src/CompletionProvider.ts

```diff
@@ -86,6 +88,16 @@ export class CSSModuleCompletionProvider implements CompletionItemProvider {
         if (this._classTransformer) {
           name = this._classTransformer(name);
         }
+        // handle kebab-case classname
+        if (name.includes('-')) {
+          const completionItem = new CompletionItem(name, CompletionItemKind.Variable);
```

Comment:
Extract a function here will make it more clear I think. Like `createBracketCompletionItem`?

Reply:
OK, thanks for suggestion